cmake/configure/configure_2_trilinos.cmake

```diff
@@ -171,7 +171,7 @@ MACRO(FEATURE_TRILINOS_FIND_EXTERNAL var)
       #
       # Check for modules.
       #
-      FOREACH(_optional_module EpetraExt ROL Sacado Tpetra MueLu Zoltan)
+      FOREACH(_optional_module EpetraExt MueLu ROL Sacado SEACAS Tpetra Zoltan)
```

Comment:
Does this mean that SEACAS is not a *required* component of Trilinos? Do we need to adjust things such as candi and our installation documentation, or is SEACAS a default component of Trilinos?